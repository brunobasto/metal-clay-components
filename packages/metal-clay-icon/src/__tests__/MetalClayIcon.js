'use strict';

import MetalClayIcon from '../MetalClayIcon';

const spritemap = '../node_modules/lexicon-ux/build/images/icons/icons.svg';

let clayIcon;

describe('MetalClayIcon', function() {
	afterEach(() => {
		if (clayIcon) {
			clayIcon.dispose();
		}
	});

	it('should generate markup for icon `add-cell`', function() {
		clayIcon = new MetalClayIcon(
			{
				spritemap: spritemap,
				symbol: 'add-foo'
			}
		);

                console.log(clayIcon.element.outerHTML);
		expect(clayIcon.element.outerHTML).toMatchSnapshot();
	});
});
