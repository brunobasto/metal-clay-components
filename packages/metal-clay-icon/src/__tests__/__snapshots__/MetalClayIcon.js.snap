// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MetalClayIcon should generate markup for icon \`add-cell\` 1`] = `"<div><svg aria-hidden=\\"true\\" class=\\"lexicon-icon lexicon-icon-add-cell\\" key=\\"undefined\\"><use xlink:href=\\"../node_modules/lexicon-ux/build/images/icons/icons.svg#add-cell\\" key=\\"undefined\\"></use></svg></div>"`;
